packages/ontario-frontend-cli/commands/ontario-add-package/handleAddPackage.js

```diff
@@ -0,0 +1,43 @@
+const path = require('path');
+const fs = require('fs');
+const { PACKAGES_CONFIG } = require('../../core/config');
+const { installPackages } = require('../../core/operations');
+const logger = require('../../core/utils/logger');
+const { handlePackageCopy } = require('../../core/utils/process/copyPackage');
+
+async function handleAddOntarioPackageCommand(cmd = {}) {
+  try {
+    switch (cmd) {
+      case 'eslint':
+        logger.info('eslint selected');
+
+        await installPackages(PACKAGES_CONFIG[cmd]?.packages, true);
+
+        if (fs.existsSync(".eslintrc.js"))
+          logger.warning(".eslint.js file already present. Please add \"extends\": \"@ongov/eslint-config-ontario-frontend\" to your existing file.");
+
+        await handlePackageCopy(path.resolve(process.cwd()), 'eslint');
+        break;
+      case 'prettier':
+        logger.info('prettier selected');
+
+        await installPackages(PACKAGES_CONFIG[cmd]?.packages, true);
+
+        if (fs.existsSync(".prettierrc.js"))
+          logger.warning(".prettierrc.js file already present. Please add \"...@ongov/prettier-config-ontario-frontend\" to your existing file.");
+
+        if (fs.existsSync(".prettierignore"))
+          logger.warning(".prettierignore file already present. Please ignore the following directories and files: node_modules/, dist/, src/assets/vendor/* and *.njk");
+
+        await handlePackageCopy(path.resolve(process.cwd()), 'prettier');
```

Comment:
```suggestion
        await handlePackageCopy(path.resolve(process.cwd()), PACKAGES_CONFIG[cwd]);
```
Then in copyPackage we could have just:
```js
async function handlePackageCopy(outputPath, package) {
  const packageFiles = package?.configFiles; // Get our packageFiles from the config
  ...
}
```

We would have to make this change for all spots that invoke handlePackageCopy, but I think it would do two things for us:
- Remove a hardcoded value
- Focus the responsibility of handlePackageCopy to not have to do a lookup for the package as well.

Thoughts?

Reply:
I am saving this as a TODO item based on our other conversations